src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -5,19 +5,17 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ImageHandler : ViewHandler<IImage, FrameworkElement>
+	public partial class ImageHandler : ViewHandler<IImage, Image>
```

Comment:
With this PR, I can now use the native view type directly, which is good.

Reply:
Nice!